multiplayer/src/components/ArcadeSimulator.tsx

```diff
@@ -9,7 +9,8 @@ let builtSimJsInfo: Promise<pxtc.BuiltSimJsInfo> | undefined;
 
 export default function Render() {
     const { state } = useContext(AppStateContext);
-    const { gameId, playerSlot, gameState, clientRole } = state;
+    const { playerSlot, gameState, clientRole } = state;
+    const gameId = state.gameState?.gameId;
```

Comment:
```suggestion
    const { gameId } = gameState;
```
minor nit.

Reply:
This doesn't work since gameState can be undefined.